core/src/main/java/fr/sncf/osrd/infra/implementation/tracks/undirected/TrackSectionImpl.java

```diff
@@ -104,7 +104,7 @@ public ImmutableRangeMap<Double, LoadingGaugeConstraint> getLoadingGaugeConstrai
         return loadingGaugeConstraints;
     }
 
-    public RangeMap<Double, Set<Integer>> getVoltages() {
+    public RangeMap<Double, String> getVoltages() {
```

Comment:
I guess it implies catenary sections cannot overlap; it makes total sense that it shouldn't happen (even though we should probably check with railway folks), but then what do we do when it does, raise a warning? (I think that's the way we should go)

Maybe it's time to rethink how to implement electrification, and make it a direct attribute of track sections. This way, we can easily detect / resolve overlaps during edition.

Reply:
Indeed having electrification as a direct track section attribute seems a better idea. However, changing that will break track section edtion so I prefer doing that change later